### PR TITLE
More Siemens Devices

### PR DESCRIPTION
**Pull request by egekorkan**
Also a gateway is now available to transform multiple protocols to HTTP

### CODE REVIEW

**Comment by egekorkan** [comment]
Call of today: no objections, I will fix the merge conflicts and merge async

File: events/2024.11.Munich/Readme.md [hunk 1]
```diff
@@ -18,7 +18,8 @@ Stakeholders from related SDOs are welcome to contribute.
...
 | Siemens      | Radiator Valve Actuator                       |  x       |        |         | KNX IoT            |                                                                    | Table 1                                                      | @mkovatsc |
 | Siemens      | Room Controller Unit                          |  x       |        |         | KNX IoT            |                                                                    | Table 1                                                      | @mkovatsc |
-| TU Munich    | Robot arm                                     |  x       |        |         | HTTP               | 2m high, 4m width; LAN/Wifi (no Internet); should be protected     | has own table                                                      |   @FadySalama |
+| Siemens      | Temperature, Humidity and CO2 Sensor (Dev. version) |  x       |        |   | HTTP               | WiFi                                                               | Table NEED                                                   | Haoyu Ren, Darko Anicic, @mahdanoura|
```
Comment (TallTed) [inline events/2024.11.Munich/Readme.md:21]:
```suggestion
| Siemens      | Temperature, Humidity, and CO2 Sensor (Dev. version) |  x       |        |   | HTTP               | WiFi                                                               | Table NEED                                                   | Haoyu Ren, Darko Anicic, @mahdanoura|
```

Comment (egekorkan) [reply events/2024.11.Munich/Readme.md]: doing it in a new commit, thanks!

File: events/2024.11.Munich/Readme.md [hunk 2]
```diff
@@ -44,6 +44,7 @@ Network:
 | Siemens      | Radiator Valve Actuator                       |  x       |        |         | KNX IoT            |                                                                    | Table 1                                                      | @mkovatsc | |
 | Siemens      | Room Controller Unit                          |  x       |        |         | KNX IoT            |                                                                    | Table 1                                                      | @mkovatsc | |
 | Siemens      | S7-1500 connected with conveyor belt and lamp |  x       |        |         | OPC UA             | LAN                                                                | Table 1                                                      | @sebastiankb | |
+| Siemens      | Temperature, Humidity and CO2 Sensor (Dev. version) |  x       |        |   | HTTP               | WiFi                                                               | Table NEED                                                   | Haoyu Ren, Darko Anicic, @mahdanoura| |
```
Comment (egekorkan) [inline events/2024.11.Munich/Readme.md:47]:
```suggestion
| Siemens      | Temperature, Humidity, and CO2 Sensor (Dev. version) |  x       |        |   | HTTP               | WiFi                                                               | Table NEED                                                   | Haoyu Ren, Darko Anicic, @mahdanoura| |
```

**Pull request merged**